### ISSUE TITLE
Statistics per Country/ASN

### ISSUE THREAD

**Issue by lasseh** [lasseh/whynoipv6#2]
Generate statistics per country and ASN.

**Comment by AndriusCTR** [comment]
@lasseh is it possible to get a full list of websites that do support IPV6? Is this data available somewhere? Thanks!

**Comment by lasseh** [comment]
@AndriusCTR This will be available in the new version of the site ( the current code of this repo ) You can check out my dev site here: https://ipv6.fail/country/SE?filter=heroes
Is this the data you are looking for? (The country mapping is not optimal atm, working on a new code for this)

**Comment by AndriusCTR** [comment]
@lasseh Yes, this is it, great! Do you have it in any structured format? json or raw list? I am specifically interested in this: https://ipv6.fail/country/US?filter=heroes

**Comment by lasseh** [comment]
Yes, you can use the API (docs for this will come later) https://ipv6.fail/api/country/US/heroes?offset=100
The result is limited to 100 per page. but if you want I can look into generating a complete list to download.

I'm currently rewriting the status fields for the domains, so there will be some breaking changes if you write a scraper for this now.

**Comment by AndriusCTR** [comment]
@lasseh Thanks, yes ability to download full list would be useful, especially when you want to do the full data export. Otherwise you would have to load hundreds / thousands pages, as each one has 50 results per page.

**Comment by lasseh** [comment]
@AndriusCTR What kind of format would you like it in? Just a full JSON dump like the api?

Can i ask what you are using it for? Great that more people are interested in this!

**Comment by AndriusCTR** [comment]
@lasseh any structured format works, could be json too. Thanks!

I am using it for custom reporting and analysis, so I need raw data. And in my case it will be used in combination with the other data to get certain market insights.
